etcdmgmt/etcd-mgmt.go

```diff
@@ -160,3 +147,115 @@ func isETCDStartNeeded() (bool, int) {
 	}
 	return start, pid
 }
+
+func EtcdStartInit() (*os.Process, error) {
+	// Check whether etcd environment variable present or not
+	// If it present then start etcd without --initial-cluster flag
+	// other wise start etcd normally.
+
+	file, err := os.Open(etcdConfFile)
+	if err != nil {
```

Comment:
Shouldn't you need to consider that if the file doesn't exist then only you have to start etcd in standalone mode? What I mean here is os.Open() can fail because of many reasons, we'd need consider only the error when the file doesn't exist and then proceed for a standalone boot otherwise its a fatal case and etcd can't be started.